core/src/main/java/org/mini2dx/connect4/Tile.java

```diff
@@ -0,0 +1,95 @@
+package org.mini2dx.connect4;
+
+
+import com.badlogic.gdx.graphics.Texture;
+import org.mini2Dx.core.graphics.Graphics;
+import org.mini2Dx.core.graphics.Sprite;
+
+public class Tile {
+
+    private int x, y; //co-oridinates of the tile.
+    private TileColour colour;
+    private static final String EMPTY_TILE = "tileGrey_27.png";
+    private static final String RED_TILE = "tileRed_27.png";
+    private static final String BLUE_TILE = "tileBlue_27.png";
+    private Sprite tileSprite;
+
+
+    public enum TileColour
+    {
+        EMPTY,
+        BLUE,
+        RED
+    }
+
+    public Tile(TileColour colour)
+    {
+        this.setColour(colour);
+    }
+
+
+    public Tile(int x, int y)
+    {
+        this.setColour(TileColour.EMPTY);
+        this.x = x;
+        this.y = y;
+
+    }
+
+
+   public Tile clone() {
+        return new Tile(this.colour);
+    }
+
+    public void render(Graphics g) {
+        g.drawSprite(tileSprite);
+
+    }
+
+    public void initialise() {
+        tileSprite.setPosition(getX(),getY());
+    }
+
+    public void update(float delta) {
+        tileSprite.setPosition(getX(),getY());
+    }
+
+    public int getX() {
+        return x;
+    }
+
+    public int getY() {
+        return y;
+    }
+
+    public TileColour getColour() {
+        return colour;
+    }
+
+    public void setX(int x) {
+        this.x = x;
+    }
+
+    public void setY(int y) {
+        this.y = y;
+    }
+
+    public void setColour(TileColour colour) {
+        this.colour = colour;
+
+        switch (colour) {
+            case RED:
+                tileSprite = new Sprite(new Texture(RED_TILE));
```

Comment:
Tiles should share texture instances to avoid loading a new texture to the graphics card for each tile.